extension/js/common/api/email-provider/gmail/gmail.ts

```diff
@@ -136,7 +136,10 @@ export class Gmail extends EmailProviderApi implements EmailProviderInterface {
   };
 
   public msgsGet = async (msgIds: string[], format: GmailResponseFormat): Promise<GmailRes.GmailMsg[]> => {
-    return await Promise.all(msgIds.map(id => this.msgGet(id, format)));
+    return await promiseAllWithLimit(
+      10,
```

Comment:
According to Gmail usage limits it's possible to have 50 concurrent `msg.get` requests - https://developers.google.com/gmail/api/reference/quota
What about increasing current 10 requests limit to 30?

Reply:
Sounds good!